Scenes/Player/Player.gd

```diff
@@ -4,7 +4,10 @@ export (int) var speed = 200
 export (PackedScene) var ps_bomb
 
 var bombs = 2;
+var bomb_range = 1
+var radious_range_const = 64
```

Comment:
To jako exportowane wartości warto zrobić, jeśli będziemy zmieniać wygląd i wielkość pól